Makefile

```diff
@@ -20,5 +24,22 @@ test: .test.venv
 integration: .integration.venv
 	$(TENV_PYTHON) -m mypy tests/integration/mypy_test.py
 
+lint: .test.venv .lint
+.lint: $(SRCS) $(TSCS)
+	$(VENV_PYTHON) -m flake8 $?
+	touch $@
+
+static: .test.venv .static
+.static: $(SRCS) $(TSCS)
+	echo "Code: $(SRCS)"
+	echo "Test: $(TSCS)"
```

Comment:
I suggest you remove this debugging code. Make will show all files test on the line below. The `$^` below is expended to include all the files tested.

```suggestion
```